joplin/users/fixtures/administrative/editor_for_test_env.py

```diff
@@ -20,6 +21,7 @@ def editor_for_test_env():
     }
 
     editor_group = Group.objects.get(name="Editors")
+    add_editor_permissions(editor_group)
```

Comment:
here is where i add the permissions to the editor fixture. I'm seeing now that I need to find a place to run the "add_editor_permissions" to the groups when they are initially made. Aren't the groups made automatically by wagtail?

Reply:
Yep they are created automatically by Wagtail. But I think permission could be added to them after the fact within a migration, like how you're adding page view restriction in the other migration file.